components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/webnotifications/GeckoWebNotificationDelegateTest.kt

```diff
@@ -1,47 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We're removing these tests for the Kotlin update?

Reply:
At first I wanted to fix them since using a null value for a non-null variable in Kotlin now throws `NullPointerException` instead of `IllegalStateException` as before, ... then I noticed that they do not actually test anything:

What happens is that `GeckoViewWebNotification` is mocked (so everything returns null). Once we pass it into `onShowNotification()` the code will call `toWebNotification()` on the mock. This will throw since `tag` is not nullable, but that's what the mock returns for `tag`. Then back in the test we catch that exception and do an assert on the message. So we do not really test our code at all. What we are testing is that the Kotlin compiler adds a null check with a specific message, a good test, but now for us? :)

Reply:
That sounds quite reasonable, thanks!